pkg/extensions/sync/sync_test.go

```diff
@@ -257,7 +257,7 @@ func TestSyncOnDemand(t *testing.T) {
 		}
 
 		syncConfig := &sync.Config{
-			Registries: []sync.RegistryConfig{syncRegistryConfig},
+			Registries: []sync.RegistryConfig{syncRegistryConfig}, Enable: true,
```

Comment:
Is there a test case, where "Enable=false" and sync is ignored?
How about for other extensions? "search", "cve"?

Reply:
Added a new test case when sync is disabled.
For the rest of extensions, they already have option to enable/disable their respective features.